libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/Microsoft.Bot.Builder.Integration.AspNet.Core.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>
```

Comment:
>netstandard2.0;netcoreapp3.0< [](start = 22, length = 29)

I think this should be <TargetFrameworks>netcoreapp2.1;netcoreapp3.0</TargetFrameworks>